### ISSUE TITLE
csutil.cxx reverseword_utf without calling table routines

### ISSUE THREAD

**Issue by phajdan** [hunspell/hunspell#294]
Looking at reverseword_utf routine, it calls 2 other routines to 1st store values in a table, then retrieve values after reversing.

Recoded routine to use a temporary utf character buffer so now it should work with 4char utf values correctly.

grep shows several locations where reverse is called, but I have not managed to find a way to quickly/easily test all possible combinations to verify this routine works.... unfortunately, do not know enough about hunspell to test this further.
In summary, routine compiles okay, but routine needs testing!

Original comment by: josedasilva

Original Ticket: "hunspell/patches/10":https://sourceforge.net/p/hunspell/patches/10

**Comment by dimztimz** [comment]
Closing this, we don't need it. Unicode decoding and encoding stuff will be completely rewritten in v2 to be both more abstract, more readable, and to be faster.
